### ISSUE TITLE
List of compatible themes

### ISSUE THREAD

**Issue by tenorslowworm** [mgsisk/webcomic#319]
Could you list some themes that you know are compatible with the universal integration method, as well as any you might know of that were written specifically for webcomic? I've tried several generic themes, and while some of them successfully show the comic for a given post, none of them so far are managing to show the latest strip on the front page of the blog. (I do have the blog set to show the latest posts rather than a specified page.)

It would help to be able to try with a theme that's known to work, so I can see if it's something else I'm doing or not doing that's the problem.

Thanks!

p.s. It might be a nice feature for your site, too, to have a page listing known compatible themes. Especially now that Wordpress is refusing to list themes that require plugins. I've tried googling for such themes, but the problem is that "webcomic" is so generic (as opposed to, say, "comic easel") that I get too many irrelevant hits.

**Comment by mgsisk** [comment]
Hey @tenorslowworm, thanks for the feedback! A list of compatible themes (and what integration methods they actually support) would be a great addition to the documentation, though I'd be concerned about maintaining such a list. If we could narrow it down to a set of "generally popular amongst comic artist" themes that might help… something to think about.

As far as I know, there's really only one theme designed specifically for Webcomic – [Inkblot](https://wordpress.org/themes/inkblot/). As you can see from that listing, it's a bit out of date, but still works perfectly well (to my knowledge, at least). I'm working on another theme for Webcomic 5+ (as well as updating Inkblot for Webcomic 5+), but that's the only option I'm aware of right now.

At the moment, the only themes I've actually tested with the Universal integration method are the "core" WordPress themes (Twenty Ten through Twenty Seventeen). The default CSS on Twenty Twelve and Twenty Thirteen aren't ideal for comic display and might need some tweaking, but the Universal integration works – including displaying comic posts on the front page – in all of those themes.

Can you provide links to some of the themes you tried (and possibly what other plugins you have running besides Webcomic)? Universal integration really ought to work with the vast majority of themes; the comic might not be exactly where you expect (or want) it or display optimally (as with the Twenty Twelve/Thirteen themes), but to completely fail makes it sound like:

1. I screwed something up, which is most likely;
2. The theme is doing something very strange, which – while not unheard of – is much less likely;
3. There's a plugin doing something very strange, which is possible (but I'd still want to check before I blamed the plugin)

**Comment by tenorslowworm** [comment]
Thanks! As it turns out, it was something I did, though I don't really know what it was.

Your reporting that it should work fine with the Wordpress "Twenty-NN" themes made me realize this, because I had tried it with them and had no success. But I hadn't really done anything with the blog yet, except try out various themes and plugins, and I still had problems if I deactivated every other plugin. So then I thought that maybe it was some peculiarity of my hosting plan, which was the Dreamhost "DreamPress" plan, and tried it on my regular Dreamhost VPS instead -- and it worked! However, when I destroyed my DreamPress instance and created a new one, it worked, too!

So it was something I did to that original installation, but I might never know what it was. I didn't do much, so I could conceivably retry everything just to narrow it down -- for example, I did have Comic Easel installed but deactivated, but maybe it did something while it was active that ended up blocking Webcomic even after it was deactivated? -- but I don't know if I have the dedication to track it down.

In any case, thanks again for your quick response. If I decide to try isolating the problem and succeed, I'll let you know.

Meanwhile, I'm excited about Inkblot and the new theme you're working on. I actually tried to use Inkblot in my first go around, and never found anything I liked better visually. Is there a feed I can follow to find out when it's ready for Webcomic5?
